_data/master/navbars/usage.yml

```diff
@@ -11,7 +11,7 @@ toc:
     path: /usage/calicoctl/install-and-configuration
   - title: calico/ctl container
     path: /usage/calicoctl/container
-- title: Calico and Systemd
+- title: Running Calico as a system service
```

Comment:
"Running Calico as a _systemd_ service" might be more accurate?

Reply:
I know what you mean, to me, it resonates more "systemd"...but in the text, it says you don't need to use systemd, you can use "init daemons such as upstart". I do not have a strong opinion on this but was just trying to stay in sync with the doc. To me, "systemd" resonates more but WDYT

Reply:
Maybe we just need to call it "Running Calico with an init system"

Reply:
We _could_ do that, but I had started to lean heavily towards "systemd", I did a little Googling and I think that systemd has achieved prominence. Apparently, there were some wars on this, but that was back in _2014_.... https://www.tecmint.com/systemd-replaces-init-in-linux/ But I am flexible on this, let me know your thoughts.